examples/common/bgfx_utils.cpp

```diff
@@ -592,7 +592,7 @@ void Mesh::submit(bgfx::ViewId _id, bgfx::ProgramHandle _program, const float* _
 		
 		bgfx::setIndexBuffer(group.m_ibh);
 		bgfx::setVertexBuffer(0, group.m_vbh);
-		bgfx::submit(_id, _program, 0, it != itEnd-1);
+		bgfx::submit(_id, _program, 0, (it == itEnd-1)?BGFX_DISCARD_ALL:BGFX_DISCARD_NONE);
```

Comment:
`(it == itEnd-1) ? BGFX_DISCARD_ALL : BGFX_DISCARD_NONE);`

Instead `BGFX_DISCARD_ALL` it should be `BGFX_DISCARD_INDEX_BUFFER|BGFX_DISCARD_VERTEX_BUFFER`